docs/view_components/create_scenario.rst

```diff
@@ -0,0 +1,54 @@
+..
+    one can then cross link to this view component by using
+    :ref:`create_scenario-label`
+
+Create Scenario
+---------------
+
+Create scenario view-component enables the user to create a new scenario object where the energy system can be implemented.
+
+Attributes
+^^^^^^^^^^
+
+**Combination of a text field and other GUI elements**
```

Comment:
This is not an exhaustive and univoque descriptions, to me this is as clear as "Some things to do stuff". Which GUI elements, please be specific. 

I think that we would need a unique id for each scenario, this could be an attribute.

Reply:
Why I did not go for specific GUI components is that I was afraid that I am describing my preferred way of rendering :/

Reply:
There is a difference between saying "there should be a button to perform this action" and "There should be a green button with font size 20 and font color magenta, placed 13 pixels on the right of the text input field with "Click me" as label". You can say something like the first sentence which mainly focus on usability and not rendering